### DISCUSSION TITLE
[Megadrive] Golden Axe (World) (Rev A)

### DISCUSSION THREAD

**Discussion by 7ank0v1c** [sanni/cartreader#696] [Need help]
Heya,

So I have just dumped a load of my carts in efforts to upload my findings on No-Intro.

Howvever, upon dumping Golden Axe (World) (Rev A) for mega drive, I noticed a weird issue.

Basically it seems either the header was not read correctly or something else is an issue, yet the game is read fine (hashes check out to be correct)

Here is my dump log text:

--------------------------------------------------------------------------------------------------------------------------------------
OSCR HW3 V12.0

[+] Mega Drive

[+] Game Cartridge

Cart Info
 
Name: fnkcdm`wd
Size: 4 MBit
ChkS: C17B
Sram: None
 
Press Button...

[+] Read Rom

Saving to MD/ROM/fnkcdm`wd/198/...
[*******************]
Internal checksum...OK
CRC32... 665D7DF9 -> Golden Axe (World) (Rev A).md

-----------------------------------------------------------------------------------------------------------------------------------------------------------

As you can see, the game header was read as _**fnkcdm`wd**_, which is very strange?

Any ideas what is happening?

I have cleaned the pins many times with isopropyl and with a pink eraser and I keep getting the same header

Here is also the photos in case it helps:

![IMG_6102](https://user-images.githubusercontent.com/40895945/216220086-a3e669a0-22ae-46b9-ad42-e38780cf4bc1.JPG)
![IMG_6101](https://user-images.githubusercontent.com/40895945/216220075-ac1b4b63-1e43-4cfe-8692-8cd00db26bfe.JPG)
![IMG_6104](https://user-images.githubusercontent.com/40895945/216220099-fa55eeb9-ed7a-4074-ad50-69eb42a4ad3f.JPG)
![IMG_6103](https://user-images.githubusercontent.com/40895945/216220093-848fff43-2daa-4877-85ae-baf0b79ae6b3.JPG)
![IMG_6100](https://user-images.githubusercontent.com/40895945/216220064-2c7eba0c-8d0d-45de-b7c5-570915d22203.JPG)

**Comment by sanni** [comment]
It's Shift-JIS and means GOLDEN AXE.

![cscssccscs](https://user-images.githubusercontent.com/19169494/216266179-8d2e2ace-9cc5-4660-87df-c4322fff0aff.jpg)
![dsdsddsdsds](https://user-images.githubusercontent.com/19169494/216266186-7c972418-e2a6-4ca4-a856-30bc4ba50d2f.jpg)

**Reply by 7ank0v1c** [reply]
So is this working correctly then?

**Reply by 7ank0v1c** [reply]
And out of interest, what’s ‘Shift-JIS’?

**Reply by 7ank0v1c** [reply]
Never mind, I done a little reading into this and can see its all good.

Thanks for the pointer =D
